content/en/training/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Training
+menu: { main: { weight: 45 } }
+description:
+  Training programs, certifications, and support resources for OpenTelemetry
```

Comment:
The page doesn't contain support resources yet so I'd either add a section ~ "coming soon" or remove that part from the title. We can always add it later. WDYT?

Reply:
Removed support resources altogether and added a coming soon for training programs in 272f315.